components/nav.php

```diff
@@ -30,7 +30,23 @@
 					<a class="nav-link" href="/queue">Queue</a>
 				</li>
 				<li class="nav-item">
+				<?php if ($USER->isLoggedIn()): ?>
```

Comment:
Also did we figure out why this PHP will fail on the Mac's MAMP?

Reply:
Perhaps it used to be shorthand tags which was an issue for some?

Reply:
Update, got this fixed by rearranging some of the `require_once` in user.class.php and it ended up working 🤷‍♂️